RDKB-58414: Dynamically update NAS

Srijeyarankesh opened the pull request:
Impacted Platforms:
All OneWifi Platforms

Reason for change:We must dynamically update NAS IP to be in accordance with RADIUS IP Family.

Test Procedure: Enable Hotspot VAPs and connect clients. Simultaneously take erouter0 PCAPs. Check if NAS IP is updated properly as per RADIUS IP Family.

Risks: Low

Change-Id: I8f33dc2d45bef2dc10581c88d6b812a49f02dd9d
 Signed-off-by:[Srijeyarankesh_JS@comcast.com]

Commit:
RDKB-58414: Dynamically update NAS

Impacted Platforms:
All OneWifi Platforms

Reason for change:We must dynamically update NAS IP to be in accordance
with RADIUS IP Family.

Test Procedure: Enable Hotspot VAPs and connect clients. Simultaneously
take erouter0 PCAPs. Check if NAS IP is updated properly as per RADIUS
IP Family.

Risks: Low

Change-Id: I8f33dc2d45bef2dc10581c88d6b812a49f02dd9d
Signed-off-by:Srijeyarankesh_JS@comcast.com

Commit:
RDKB-58414: Dynamically update NAS

Impacted Platforms:
All OneWifi Platforms

Reason for change:We must dynamically update NAS IP to be in accordance
with RADIUS IP Family.

Test Procedure: Enable Hotspot VAPs and connect clients. Simultaneously
take erouter0 PCAPs. Check if NAS IP is updated properly as per RADIUS
IP Family.

Risks: Low

Change-Id: I8f33dc2d45bef2dc10581c88d6b812a49f02dd9d
Signed-off-by:Srijeyarankesh_JS@comcast.com

Srijeyarankesh committed:
RDKB-58414: Dynamically update NAS

Impacted Platforms:
All OneWifi Platforms

Reason for change:We must dynamically update NAS IP to be in accordance with RADIUS IP Family.

Test Procedure: Enable Hotspot VAPs and connect clients. Simultaneously take erouter0 PCAPs. Check if NAS IP is updated properly as per RADIUS IP Family.

Risks: Low

Change-Id: I8f33dc2d45bef2dc10581c88d6b812a49f02dd9d
 Signed-off-by:Srijeyarankesh_JS@comcast.com

Srijeyarankesh committed:
Merge branch 'main' of github.com:Srijeyarankesh/hostap-patches into main

Srijeyarankesh committed:
Merge branch 'main' of github.com:Srijeyarankesh/hostap-patches into main

Srijeyarankesh committed:
Merge branch 'main' of github.com:Srijeyarankesh/hostap-patches into main

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/hostap-patches?pullRequest=11) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/hostap-patches?pullRequest=11) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Srijeyarankesh<br/>:x: sjs889<br/><hr/>**sjs889** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/hostap-patches?pullRequest=11) it.</sub>

Srijeyarankesh committed:
RDKB-58414: Dynamically update NAS

Impacted Platforms:
All OneWifi Platforms

Reason for change:We must dynamically update NAS IP to be in accordance
with RADIUS IP Family.

Test Procedure: Enable Hotspot VAPs and connect clients. Simultaneously
take erouter0 PCAPs. Check if NAS IP is updated properly as per RADIUS
IP Family.

Risks: Low

Change-Id: I8f33dc2d45bef2dc10581c88d6b812a49f02dd9d
Signed-off-by:Srijeyarankesh_JS@comcast.com
